keysign/gpgmeh.py

```diff
@@ -530,8 +536,22 @@ def import_signature(encrypted_sig, homedir=None):
         log.warning("The signature that we were importing is not as we expected!")
```

Comment:
Apparently this was an existing bug, but now that you are changing this function maybe you can also fix that in this same PR.

In the line above, `result.revocations` needs to be changed to `result.new_revocations` because `ImportResult` object has no `revocations` attribute.